### ISSUE TITLE
Test epoch should be 600?

### ISSUE THREAD

**Issue by csyanbin** [crashmoon/MAML-Pytorch-Multi-GPUs#1]
Hi, thanks for the code implementation.
This codebase is really of high quality and neat to read. 

When I ran the default param, I can reproduce 48.9%, however, I found two issues:
1. the test epoch is set to 25 here: https://github.com/crashmoon/MAML-Pytorch-Multi-GPUs/blob/master/train_mini_adam.py#L116. The common epoch is 600 fir MAML, with this param, I only got 44.60%.
2. you are testing the best iteration on test set directly. I think this should be on the validation set, and apply the best iteration to test set.
